### PR TITLE
Schema: Remove mixed types: road.surface (Parquet compatibility)

### PR DESCRIPTION
**Pull request by TheRBajaj**
# Description

This PR removes the mixed schema types for `roads.surface` property

### Before
`roads.surface` could accept multiple types of values that could be put into the same property like array and string values.

### After

`roads.surface` only accepts array property instead of array and string

# Reference

https://github.com/OvertureMaps/tf-transportation/issues/57

# Testing

Tested using test script

### CODE REVIEW

File: schema/transportation/segment.yaml [hunk 1]
```diff
@@ -147,18 +147,16 @@ properties:
           description: Physical surface of the road. May either be
             specified as a single global value for the segment, or as
             an array of surface rules.
-          oneOf:
```
Comment (sanjibdutta1) [inline schema/transportation/segment.yaml:150]: don't we require similar fix for flags?

Comment (TheRBajaj) [reply schema/transportation/segment.yaml:150]: yup I will have another PR out for the flags fix too. We decided to break this issue up into multiple PR's

**Review by brad-richardson — DISMISSED** [review]
Nice!

File: schema/transportation/segment.yaml [hunk 2]
```diff
@@ -147,18 +147,16 @@ properties:
...
+          items:
+            type: object
+            allOf:
+              - { "$ref": "../defs.yaml#/$defs/propertyContainers/atRangeContainer" }
+            unevaluatedProperties: false
+            properties:
+              value: { "$ref": "#/$defs/propertyDefinitions/roadSurface" }
+          minItems: 1
+          uniqueItems: true
           default:
```
Comment (vcschapp) [inline schema/transportation/segment.yaml:160]: This `default` value makes no sense now - can you delete it?

**Review by vcschapp — CHANGES_REQUESTED** [review]
Thanks Rahul. Couple small things:

1. [x] The `default` value should be deleted.
2. [x] The counterexample [`bad-road-surface-invalid-flat-value.json`](https://github.com/OvertureMaps/schema/blob/18023ca39cbee55df604897d8d22689dce54f574/counterexamples/transportation/segment/road/bad-road-surface-invalid-flat-value.json) is no longer applicable and should be deleted.

**Review by marinrelatic — APPROVED** [review]

**Review by brad-richardson — APPROVED** [review]

**Review by vcschapp — APPROVED** [review]

**Review by sanjibdutta1 — APPROVED** [review]

**Pull request merged**